multi-container/hello-nginx-sample/README.md

```diff
@@ -0,0 +1,67 @@
+# Deploy simple nginx multi-container service
+
+A Google Cloud Project is required in order to run the sample. The project should have the following API's enabled:
+
+* Cloud Run
+* Secret Manager
+
+## Enable required APIs
+
+```bash
+gcloud services enable secretmanager.googleapis.com run.googleapis.com
+```
+
+## Add nginx server code in a secret with Secret Manager
+
+Utilizing [Secret Manager](https://cloud.google.com/secret-manager) to store and mount our [nginx](https://www.nginx.com/) server code.
+
```

Comment:
Add something to the effect of: "Instead of packaging the nginx config into the container image, the config will be mounted as a volume at runtime. This allows for separation of config from code."